cluster/gce/manifests/glbc.manifest

```diff
@@ -1,20 +1,19 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: l7-lb-controller-v1.2.3
+  name: l7-lb-controller
   namespace: kube-system
   annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ''
     seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
   labels:
     k8s-app: gcp-lb-controller
-    version: v1.2.3
     kubernetes.io/name: "GLBC"
 spec:
-  priorityClassName: system-node-critical
   terminationGracePeriodSeconds: 600
   hostNetwork: true
   containers:
-  - image: k8s.gcr.io/ingress-gce-glbc-amd64:v1.2.3
+  - image: k8s.gcr.io/ingress-gce-glbc-amd-64:v1.6.1
```

Comment:
s/amd-64/amd64/

With that I hope tests should pass.

Reply:
Done